src/app.vue

```diff
@@ -1,16 +1,14 @@
 <template>
-  <div class="flex-col d-flex flex-column align-items-center">
-    <BaseLabel>Hello World</BaseLabel>
-  </div>
+  <SignUp />
```

Comment:
Can we create `/signup` route and ensure signup experience exists on that route?